.github/workflows/add-to-backlog.yml

```
@@ -0,0 +1,20 @@
+name: Add issues to shared backlog
+
+on:
+  issues:
+    types:
+      - opened
+  pull_request:
+    branches-ignore: 
+      - "dependabot/*"
```

If I remember correctly, we needed to do this to avoid it to be run on Dependabot branches:

```suggestion
      - "dependabot/**/*"
```

---

Thanks, will update the template too!